build_tools/docker/prod_digests.txt

```diff
@@ -1,21 +1,34 @@
+# General
```

Comment:
I'd avoid having comments in this file or trying to impose a strict order. The whole point is to be able to easily script around it. These comments will break the parsing in manage_images.py. If you want to reorder it, that's alright, although given that new images will not respect the order, I'm not sure there's much value

Reply:
That would be easy to fix with something like `if len(line) and not line.startswith("#"):`. 
I've had to manually copy shas from this file enough that that line of code seems worth adding, and I don't think the sections need to be 100% accurate, just useful. I guess I'll hopefully not have to open that file for a long time anyway.

Reply:
Well at the very least it should happen in the same PR. I don't really want to reinvent comment parsing.... Can we go ahead and fix the bare docker images first and save this for later if necessary.